src/rpc/rpcevo.cpp

```diff
@@ -157,106 +165,167 @@ static std::string SignAndSendSpecialTx(const CMutableTransaction& tx)
+            "\nCreates, funds and sends a ProTx to the network. The resulting transaction will move 1000 Dash\n"
             "to the address specified by collateralAddress and will then function as the collateral of your\n"
             "masternode.\n"
             "A few of the limitations you see in the arguments are temporary and might be lifted after DIP3\n"
             "is fully deployed.\n"
             "\nArguments:\n"
             "1. \"collateralAddress\"   (string, required) The dash address to send the collateral to.\n"
             "                         Must be a P2PKH address.\n"
-            "2. \"collateralAmount\"    (numeric or string, required) The collateral amount.\n"
-            "                         Must be exactly 1000 Dash.\n"
-            "3. \"ipAndPort\"           (string, required) IP and port in the form \"IP:PORT\".\n"
+            "2. \"ipAndPort\"           (string, required) IP and port in the form \"IP:PORT\".\n"
             "                         Must be unique on the network. Can be set to 0, which will require a ProUpServTx afterwards.\n"
-            "4. \"ownerKeyAddr\"        (string, required) The owner key used for payee updates and proposal voting.\n"
+            "3. \"ownerKeyAddr\"        (string, required) The owner key used for payee updates and proposal voting.\n"
             "                         The private key belonging to this address be known in your wallet. The address must\n"
             "                         be unused and must differ from the collateralAddress\n"
-            "5. \"operatorKeyAddr\"     (string, required) The operator key address. The private key does not have to be known by your wallet.\n"
+            "4. \"operatorKeyAddr\"     (string, required) The operator key address. The private key does not have to be known by your wallet.\n"
             "                         It has to match the private key which is later used when operating the masternode.\n"
             "                         If set to \"0\" or an empty string, ownerAddr will be used.\n"
-            "6. \"votingKeyAddr\"       (string, required) The voting key address. The private key does not have to be known by your wallet.\n"
+            "5. \"votingKeyAddr\"       (string, required) The voting key address. The private key does not have to be known by your wallet.\n"
             "                         It has to match the private key which is later used when voting on proposals.\n"
             "                         If set to \"0\" or an empty string, ownerAddr will be used.\n"
-            "7. \"operatorReward\"      (numeric, required) The fraction in %% to share with the operator. If non-zero,\n"
+            "6. \"operatorReward\"      (numeric, required) The fraction in %% to share with the operator. If non-zero,\n"
```

Comment:
Is the double % necessary ("%%")? The help message displays both of them.

Reply:
can confirm this as well

Reply:
```suggestion
            "6. \"operatorReward\"      (numeric, required) The fraction in % to share with the operator. If non-zero,\n"
```